gpu-operator/release-notes.rst

```diff
@@ -31,9 +31,132 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
 
    GPU Operator beta releases are documented on `GitHub <https://github.com/NVIDIA/gpu-operator/releases>`_. NVIDIA AI Enterprise builds are not posted on GitHub.
 
-
 ----
 
+.. _v25.3.0:
+
+25.3.0
+======
+
+.. _v25.3.0-new-features:
+
+New Features
+------------
+
+* Added support for the following software component versions:
+
+  - NVIDIA Container Toolkit version v1.17.5
+  - NVIDIA Driver Manager for Kubernetes v0.8.0
+  - NVIDIA Kubernetes Device Plugin v0.17.1
+  - NVIDIA DCGM Exporter v4.1.1-4.0.4
+  - NVIDIA DCGM v4.1.1-2
+  - Node Feature Discovery v0.17.2
+  - NVIDIA GPU Feature Discovery for Kubernetes v0.17.1
+  - NVIDIA MIG Manager for Kubernetes v0.12.1
+  - NVIDIA KubeVirt GPU Device Plugin v1.3.1
+  - NVIDIA vGPU Device Manager v0.3.0
+  - NVIDIA Kata Manager for Kubernetes v0.2.3
+  - NVIDIA GDRCopy Driver v2.4.4
+
+* Added support for the NVIDIA GPU DRA Driver v25.3.0 component which enables Multi-Node NVLink through Kubernetes Dynamic Resource Allocation (DRA)  and IMEXsupport.
```

Comment:
Since the DRA driver is not officially out yet, let's add a "coming soon" 
```suggestion
* Added support for the NVIDIA GPU DRA Driver v25.3.0 component (coming soon) which enables Multi-Node NVLink through Kubernetes Dynamic Resource Allocation (DRA)  and IMEXsupport.
```

Reply:
added